tests/fixtures/diff.txt

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
as I said, you can create python module with `"""..."""` strings and pre-commit won't bother with them